.github/workflows/main.yml

```diff
@@ -0,0 +1,18 @@
+name: build
```

Comment:
Let's not add this file in this PR, I need to generate an add the coveralls token for the action so I'd rather do that in the other PR.